packages/gis/lua/Layer.lua

```diff
@@ -593,6 +605,7 @@ Layer_ = {
 	-- with each new layer's name formed by the own Layer's name and the respective times as sufix.
 	-- The default value is false and, in this case, the temporal data will be filled in the own Layer
 	-- into different attributes though.
+	-- @arg data.progress A boolean to set the progress viewer of the filling. Its default is true.
```

Comment:
A boolean value indicating whether progress will be shown while computing the operation. The default value is true.